lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,194 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
+
+    $fileLastMod = ([System.IO.FileInfo]$dest).LastWriteTimeUtc
+    $webLastMod = $null
+
+    $webRequest = [System.Net.HttpWebRequest]::Create($url)
+
+    foreach ($header in $headers.GetEnumerator()) {
+        $webRequest.Headers.Add($header.Name, $header.Value)
+    }
+
+    if ($timeout) {
+        $webRequest.Timeout = $timeout * 1000
```

Comment:
`Timeout` is not a valid property of `WebClient` https://msdn.microsoft.com/en-us/library/system.net.webclient(v=vs.110).aspx. Might be best to remove this new feature so we can get the rest in for 2.4 but I'll let you have a look.

Reply:
But this is not a WebClient object, it is a HttpWebRequest object, and it does have a Timeout property. I also see a ConnectTimeout property, which we also could add. https://msdn.microsoft.com/en-us/library/system.net.httpwebrequest(v=vs.110).aspx

Reply:
Sorry ignore this line, line 88 is the one that is the invalid property.